astropy/io/ascii/cds.py

```diff
@@ -5,24 +5,70 @@
   Classes to read CDS / Vizier table format
 
 :Copyright: Smithsonian Astrophysical Observatory (2011)
-:Author: Tom Aldcroft (aldcroft@head.cfa.harvard.edu)
+:Author: Tom Aldcroft (aldcroft@head.cfa.harvard.edu), \
+         Suyog Garg (suyog7130@gmail.com)
 """
 
 
+from io import StringIO
+
 import fnmatch
 import itertools
 import re
 import os
+import math
+import numpy as np
 from contextlib import suppress
 
 from . import core
 from . import fixedwidth
 
 from astropy.units import Unit
+from astropy import units as u
+
+from astropy.table import Table
+from astropy.table import Column, MaskedColumn
+from astropy.coordinates import SkyCoord
+from string import Template
+from textwrap import wrap, fill
+
+MAX_SIZE_README_LINE = 80
+MAX_COL_INTLIMIT = 100000
 
 
 __doctest_skip__ = ['*']
 
+cdsdicts = {'title': 'Title ?',
+            'author': '1st author ?',
+            'catalogue': '',
+            'date': 'Date ?',
+            'abstract': 'Abstract ?',
+            'authors': 'Authors ?',
+            'bibcode': 'ref ?',
+            'keywords': '',
+            'table_description': '',
+            'seealso': ''
+            }
+
+BYTE_BY_BYTE_TEMPLATE = ["Byte-by-byte Description of file: $file",
+"--------------------------------------------------------------------------------",
+" Bytes Format Units  Label     Explanations",
+"--------------------------------------------------------------------------------",
+"$bytebybyte",
+"--------------------------------------------------------------------------------"]
+
+README_TEMPLATE = os.path.dirname(os.path.realpath(__file__)) + "/src/ReadMe.template"
```

Comment:
Is there a reason for `realpath`? That's not typical. And although Python does some magic to make things work with linux-style path separators in Windows, the preferred idiom in astropy is
```
os.path.join(os.path.dirname(__file__), 'src', 'ReadMe.template')
```

Reply:
Nope, there isn't. I shall make alterations.